node/src/actor_system_factory.rs

```diff
@@ -616,29 +648,29 @@ mod tests {
             >,
         >,
         prepare_initial_messages_results: RefCell<Vec<StreamHandlerPoolSubs>>,
-        cryptdes_results: RefCell<Vec<(&'static dyn CryptDE, &'static dyn CryptDE)>>, //first main, second alias
+        cryptde_pair_results: RefCell<Vec<CryptdePair>>,
         validate_database_chain_params: Arc<Mutex<Vec<Chain>>>,
     }
 
     impl ActorSystemFactoryTools for ActorSystemFactoryToolsMock {
         fn prepare_initial_messages(
             &self,
-            main_cryptde: &'static dyn CryptDE,
-            alias_cryptde: &'static dyn CryptDE,
+            cryptdes: CryptdePair,
             config: BootstrapperConfig,
+            _persistent_config: &mut dyn PersistentConfiguration,
```

Comment:
I guess I'd like to see this Boxed and stored in the parameter list so that tests could verify that the PersistentConfiguration they sent in was the one that was used to call prepare_initial_messages().